openlibrary/core/lending.py

```diff
@@ -118,6 +119,25 @@ def is_borrowable(identifiers, acs=False, restricted=False):
     except Exception as e:
         return {'error': 'request_timeout'}
 
+def get_available(limit=None, page=1, subject=None):
+    """Retrieves a list of available works for carousels from archive.org"""
+    url = "https://archive.org/advancedsearch.php?q=collection%3Ainlibrary"
+    limit = limit or MAX_IA_LIMIT
+    if subject:
+        url += "+AND+openlibrary_subject%3A" + subject
+    url += "+AND+loans__status__status%3AAVAILABLE&fl%5B%5D=identifier&fl%5B%5D=openlibrary_edition&fl%5B%5D=openlibrary_work&&fl%5B%5D=openlibrary_edition&sort%5B%5D=&sort%5B%5D=&sort%5B%5D=&rows=" + str(limit) + "&page=" + str(page) + "&output=json"
```

Comment:
consider moving params to a dict and urlencoding to make it a little more clear/readable what's going on here